bundle/manifests/projectcontour-contour-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,350 @@
+          resources:
+          - extensionservices/status
+          - httpproxies/status
+          verbs:
+          - create
+          - get
+          - update
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          - clusterroles
+          - rolebindings
+          - roles
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - update
+          - watch
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: default
+      deployments:
+      - name: contour-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: contour-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: contour-operator
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                resources: {}
+              - args:
+                - --metrics-addr=127.0.0.1:8080
+                - --enable-leader-election
+                command:
+                - /contour-operator
+                image: docker.io/projectcontour/contour-operator:main
+                imagePullPolicy: Always
+                name: contour-operator
+                resources:
+                  requests:
+                    cpu: 100m
+                    memory: 70Mi
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps/status
+          verbs:
+          - get
+          - update
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+        serviceAccountName: default
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - projectcontour-contour-operator
+  links:
```

Comment:
I don't know whether the following fields are not set because it's a WIP or not: `links`, `maintainers`, `provider`, `version`.

Reply:
I used what was already in place. I'll update this to be more specific. I left it WIP for this very reason.